packages/sync/src/modules/class-posts.php

```diff
@@ -601,15 +616,18 @@ public function send_published( $post_ID, $post ) {
 		 */
 		$flags = apply_filters( 'jetpack_published_post_flags', $post_flags, $post );
 
-		/**
-		 * Action that gets synced when a post type gets published.
-		 *
-		 * @since 4.4.0
-		 *
-		 * @param int $post_ID
-		 * @param mixed array $flags post flags that are added to the post
-		 */
-		do_action( 'jetpack_published_post', $post_ID, $flags );
+		// Only Send Pulished Post event if post_type is not blacklisted.
+		if ( ! in_array( $post->post_type, Settings::get_setting( 'post_types_blacklist' ), true ) ) {
```

Comment:
Does it make sense to use the same `is_post_type_allowed` call here too? Or should we keep calls to `is_post_type_allowed` limited to the scope of the specific sync module?

Reply:
👍 I'll update as yes using the helper means 1 place to modify if the settings ever get re-worked.